primedev/core/convar/cvar.h

```diff
@@ -1,10 +1,11 @@
 #pragma once
 #include "convar.h"
+#include "concommand.h"
+#include "core/macros.h"
 
 //-----------------------------------------------------------------------------
 // Forward declarations
 //-----------------------------------------------------------------------------
-class ConCommandBase;
```

Comment:
I assume that class is unused and hence removed? Separate PR maybe? ^^

Reply:
the issue I had at the time is that `ConCommandBase` was used directly but I can't spot that from a glance.
Will have to test if this is actually needed.

Reply:
reverted